helloworld-mutual-ssl-secured/certs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
think this was replaced by the cli scrip, and thus should be removed

Reply:
Removed!